docs/datasets/index.rst

```diff
@@ -52,10 +52,28 @@ Note that in Gammapy, 2D image analyses are done with 3D cubes with a single
 energy bin, e.g. for modeling and fitting,
 see the `2D map analysis tutorial <./tutorials/image_analysis.html>`__.
 
-
 To analyse multiple runs, you can either stack the datasets together, or perform
 a joint fit across multiple datasets.
 
+
+Predicted counts
+================
+Predicted counts are computed like:
+
+.. math::
+
+	N_{Pred} = N_{Bkg} + \mathrm{PSF} \circledast \mathrm{EDISP(\mathcal{E} \cdot F_{Src}(l, b, E_{True}))}
+
+
+Where :math:`$N_{Bkg}$` is the expected residual hadronic background,
+:math:`$F_{Src}$` the integrated flux of the source model,
```

Comment:
Normally it should be differential no?

Reply:
Hm, the spectral model is integrated in energy and the spatial model at least integrated zero order by multiplying with the solid angle...I think one can keep the statement like this.